src/stream/src/executor/debug/trace.rs

```diff
@@ -52,10 +52,12 @@ pub async fn trace(
     while let Some(message) = input.next().instrument(span()).await.transpose()? {
         if let Message::Chunk(chunk) = &message {
             if chunk.cardinality() > 0 {
-                metrics
-                    .executor_row_count
-                    .with_label_values(&[&actor_id_string, &executor_id_string])
-                    .inc_by(chunk.cardinality() as u64);
+                if ENABLE_EXECUTOR_ROW_COUNT {
+                    metrics
+                        .executor_row_count
+                        .with_label_values(&[&actor_id_string, &executor_id_string])
+                        .inc_by(chunk.cardinality() as u64);
+                }
```

Comment:
The sampling method can be used in `executor_row_count`. Maybe implement it in the next PR.